app/cmd/cli/root.go

```diff
@@ -91,11 +91,32 @@ var startCmd = &cobra.Command{
 			comd.Env = env
 			comd.Stdin = os.Stdin
 			comd.Stdout = os.Stdout
-			_ = comd.Run()
+
```

Comment:
Curious about this, comd.Run() seems to be the same underneath as comd.Start() + comd.Wait() .